cdap-common/src/main/java/co/cask/cdap/common/io/Locations.java

```diff
@@ -341,9 +341,9 @@ public static Location getParent(Location location) {
   }
 
   /**
-   * Returns the relative path for a given location
-   * @param locationFactory
-   * @param locationURI
+   * Returns the relative path for a given location URI
```

Comment:
Should mention that the location factory must have been used to create the location URI, otherwise this just returns the uri's path.